### ISSUE TITLE
install on window set wrong alias path

### ISSUE THREAD

**Issue by xileftenurb** [rycus86/githooks#8]
Hello,

I followed the instruction for installation, on Window, using git bash with the " -- --single" options.

the installation worked, but I had to change the alias

on installation, the git hooks alias was set to "!~C:/Program Files (x86)/Git/.githooks/bin/githooks"

but the script was installed at : 
"C:/Users/**username**/.githooks/bin/githooks"

I changed manually the alias, and it worked, but it would be better if it worked from the start

thank you

**Comment by rycus86** [comment]
Hi,

Thanks a lot for reporting this!
It's very odd, I'll have a look at it.

Thanks!

**Comment by rycus86** [comment]
Sorry, this took a bit long to come back to.
Looks like the problem may have been that in Git Bash apparently `~` does not always resolve to the same directory as `$HOME` ?

**Comment by mcfandw** [comment]
Greetings!

I had been looking into this myself, because I also ran into this problem. It seems the issue is with the "!\~" combination. In addition to the version you used in c9bd7ae3, I found that separating the exclamation point from the tilde with a space would also work:
` git config --global alias.hooks '! ~/.githooks/bin/githooks' `

I don't understand why, but thought I'd pass this along.
Thank you for your work on githooks!
